### DISCUSSION TITLE
joining organisation

### DISCUSSION THREAD

**Discussion by MAHIMA-KHANDELWAL** [Amna-Hassan04/Serenity-Guide#125] [Q&A] [answered]
Hello Ma'am , I just wanted to ask that when I am trying to folk the repository , an error is coming saying I am not in any organization .
How to join the organization ma'am .

**Comment by Amna-Hassan04** [comment, marked as answer]
@MAHIMA-KHANDELWAL hey, the repository is not the part of any organization, maybe clear cache of your browser and try again, if not then do ping me again.

**Reply by MAHIMA-KHANDELWAL** [reply]
ma'am I tried many times , but nothing is happening .

**Reply by Amna-Hassan04** [reply]
plz share the screenshot here @MAHIMA-KHANDELWAL of what error your facing. @dev-shinde can u plz help her.
